pontoon/checks/utils/translate_toolkit.py

```diff
@@ -0,0 +1,78 @@
+from translate.filters import checks
+from translate.lang import data as lang_data
+from translate.storage import base as storage_base
+
+
+def quality_check(original, string, locale, disabled_checks=None):
+    """Check for obvious errors like blanks and missing interpunction."""
+    original = lang_data.normalized_unicode(original)
+    string = lang_data.normalized_unicode(string)
+    disabled_checks = disabled_checks or []
+
+    unit = storage_base.TranslationUnit(original)
+    unit.target = string
+    checker = checks.StandardChecker(
+        checkerconfig=checks.CheckerConfig(targetlanguage=locale.code),
+        excludefilters=disabled_checks
+    )
+
+    warnings = checker.run_filters(unit)
+    if warnings:
```

Comment:
Now that the diff is huge anyways, I suggest we save some indentation by turning this `if - else` block into:
```python
if not warnings:
    return {}

# The rest of the code
```